torch/optim/optimizer.py

```diff
@@ -234,7 +234,8 @@ def _get_capturable_supported_devices(supports_xla: bool = True) -> List[str]:
 
 _fused_doc = r"""fused (bool, optional): whether the fused implementation is used.
             Currently, `torch.float64`, `torch.float32`, `torch.float16`, and `torch.bfloat16`
-            are supported. (default: None)
+            are supported. A prototype implementation of Adam and AdamW for MPS supports `torch.float32` and `torch.float16`.
```

Comment:
Can you add this only to the adam/w docs and not the general fused doc? It'd a bit weird to read the SGD docs and see this line